app/routes/preprints/provider/moderation.js

```diff
@@ -56,7 +56,7 @@ export default Ember.Route.extend({
         }).then(response => {
             return {
                 submissions: response.toArray(),
-                totalPages: response.meta.total,
+                totalPages: Math.ceil(response.meta.reviews_state_counts[params.status] / 10),
```

Comment:
The pagination meta is available under `response.links.meta`, so you could do `Math.ceil(response.links.meta.total / response.links.meta.per_page)` instead